ui/packages/catalog-ui-search/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/openlayers": "4.3.11",
     "ace": "*",
     "chai": "4.1.1",
+    "inject-loader": "4.0.1",
```

Comment:
I feel like using this loader is leveraging internal implementation details. Could we not optionally pass in the metacard definitions instead?